src/qos_client/src/cli/services.rs

```diff
@@ -49,7 +49,7 @@ const DANGEROUS_DEV_BOOT_MEMBER: &str = "DANGEROUS_DEV_BOOT_MEMBER";
 const DANGEROUS_DEV_BOOT_NAMESPACE: &str =
 	"DANGEROUS_DEV_BOOT_MEMBER_NAMESPACE";
 
-#[allow(dead_code)]
+#[cfg(not(feature = "smartcard"))]
```

Comment:
Used in a bunch of places and all these places require the same `#[cfg(not(feature = "smartcard"))]`